python/pyspark/sql/readwriter.py

```diff
@@ -545,6 +545,57 @@ def partitionBy(self, *cols):
         self._jwrite = self._jwrite.partitionBy(_to_seq(self._spark._sc, cols))
         return self
 
+    @since(2.2)
+    def bucketBy(self, numBuckets, *cols):
+        """Buckets the output by the given columns on the file system.
```

Comment:
Both

```
"""
...
"""
```

or

```
"""...
"""
```

comply pep8 for multiple-line docstring up to my knowledge although I don't think a specific way has been preferred in this case.